dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/process_definition.py

```diff
@@ -100,6 +111,8 @@ def __init__(
         project: Optional[str] = configuration.WORKFLOW_PROJECT,
         tenant: Optional[str] = configuration.WORKFLOW_TENANT,
         worker_group: Optional[str] = configuration.WORKFLOW_WORKER_GROUP,
+        warning_type: Optional[str] = WarningType.NONE,
+        warning_group_id: Optional[int] = 0,
```

Comment:
could you add `warning type` to our `default_config.yaml` and then use the configuration as default value just like `configuration.WORKFLOW_WORKER_GROUP` and `configuration.WORKFLOW_TENANT` https://github.com/apache/dolphinscheduler/blob/59a026d897d5fa3599dd58e932d164370a63b5f4/dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/configuration.py#L183-L185